Sources/Classes/core/SecureStorage/Database/MiniAppSecureStorageSqliteDatabase.swift

```diff
@@ -63,10 +63,6 @@ class MiniAppSecureStorageSqliteDatabase: MiniAppSecureStorageDatabase {
     }
 
     func load(completion: ((MiniAppSecureStorageError?) -> Void)?) {
-        guard doesStoragePathExist else {
-            completion?(.storageUnavailable)
-            return
-        }
```

Comment:
Not sure why we want to remove this code? Could you please provide the reason why you want to remove it? It is expected that we will not DB file until we store some value in the DB, and it will return `storageUnavailable`

Reply:
Because, if we do not remove then on load storage call the result will be error always "storageUnavailable". so on loading the MiniApp the secure storage is not available all. Either we should stop sending the response to js-bridge on loading the mini app to avoid the behavior of blank screen and also handle the error processing In the js sample app. Then there will be no need of any code changes in the iOS SDK I believe.

Reply:
As we discussed, Secure storage ready notifications can't be removed. This seems to have an issue more on the JS Bridge or sample app. Please fix the same

Reply:
Got It, Thanks for the clarification. We will check the QA recommendation and proceed with the fix in JS-Sample app.
We are good to close this PR as these changes are not required on the iOS-SDK side.